tests/contrib/fastapi/test_fastapi.py

```diff
@@ -172,7 +172,7 @@ def test_read_item_query_string(client, tracer, test_spans):
     assert request_span.resource == "GET /items/{item_id}"
     assert request_span.error == 0
     assert request_span.get_tag("http.method") == "GET"
-    assert request_span.get_tag("http.url") == "http://testserver/items/foo"
+    assert request_span.get_tag("http.url") == "http://testserver/items/foo?q=query"
```

Comment:
This seems like a fix. We should call out the affected integrations in the release note.

Reply:
Not really, the reason for these changes in the tests is that the query string is also sent in `http.url` if `trace_query_string` is enabled, it was sent separately before.